inventory/byo/hosts.origin.example

```diff
@@ -571,10 +571,18 @@ openshift_master_identity_providers=[{'name': 'htpasswd_auth', 'login': 'true',
 # network blocks should be private and should not conflict with network blocks
 # in your infrastructure that pods may require access to. Can not be changed
 # after deployment.
+#
+# WARNING : Do not pick subnets that overlap with the default Docker bridge subnet of
+# 172.16.0.0/16.  Your installation will fail and/or your configuration change will
+# cause the Pod SDN or Cluster SDN to fail.
+#
+# WORKAROUND : If you must use an overlapping subnet, you can relocate the Docker
+# default subnet by doing the following before installing OCP:
+#     $ echo 'DOCKER_NETWORK_OPTIONS="--bip=192.168.2.1/24"' >> /etc/sysconfig/docker-network
```

Comment:
That will break things if they were already assigning to DOCKER_NETWORK_OPTIONS in that file.

Do we really need to spell out how to do this here anyway? Just add "docker's private network" to the list of things not to conflict with in the existing paragraph.